src/elements/ops.rs

```diff
@@ -537,6 +465,85 @@ pub enum Instruction {
 	TableCopy,
 }
 
+#[cfg(feature="atomics")]
+#[derive(Clone, Debug, PartialEq, Eq, Hash)]
+pub enum AtomicsInstruction {
+	AtomicWake(MemArg),
```

Comment:
This is actually not because of overlooking, but deliberate. The variants are named according to  their corresponding parts in the spec. In the spec, this instruction is named as `atomic.wake`